ci/build.sh

```diff
@@ -35,16 +35,16 @@ function compare_blob_sha_with_new_file () {
 
 function get_major_version() {
   local version="$1"
-  echo "$version" | grep -Po '^\d+'
+  echo "$version" | sed -n 's/\([0-9]*\).*/\1/p'
```

Comment:
Could you provide the context why we change the version parsing?

Reply:
Both commands yield the same result. Just that the `grep` args are not available in our Mac's `grep`, which makes it annoying to test these scripts locally from our Macs. The way I found around this was to use `sed` such a way it works on Ubuntu (CI) and Mac (locally).